spec/javascripts/google_tag_manager_url_snippet_spec.js

```diff
@@ -0,0 +1,21 @@
+describe("Google Tag Manager snippet", function () {
+
+  it("should remove params from the URL", function () {
```

Comment:
Can we get these tests to cover more of the production code?  It's currently only testing lines 9-11.